### DISCUSSION TITLE
What's next?

### DISCUSSION THREAD

**Discussion by invisal** [web3together/web3together#8] [General]
Since we have gathered plenty of people here, we should choose next step:
- Maybe we come up with one project and everyone can work together. It will help everyone learn and practice. It can be simple project.
- Regular meeting. Once every month or biweekly meeting.

Since @samnang created this group, maybe he has some ideas.

**Comment by samnang** [comment]
Thanks for raising this. I have listed some ideas about [activities](https://github.com/web3together/web3together/discussions/4) we could do regularly.

We definitely need the first meetup to know each other more and discuss the basics of topics and get some feedback on what we want to do in the group.

I agree on biweekly meeting on the topics and has some challenges to practices for the newbies to get started.
